src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using @BeforeEach for initializing the RegistrationServiceImpl instance, as it will create a new instance before each test, ensuring that tests do not interfere with each other. @BeforeAll is generally used for static initializations and should be used with static methods.